step-capstone/src/components/TravelObjectForms/AddEvent.js

```diff
@@ -134,6 +134,8 @@ export default function AddEvent(props) {
             label={"Start"}
             value={startDate}
             onChange={setStartDate}
+            error={checked && hasTimeConflict(props.data.id, startDate, endDate, props.travelObjects)}
+            helperText={checked && hasTimeConflict(props.data.id, startDate, endDate, props.travelObjects) ? "You already have something scheduled for that time" : ""}
```

Comment:
Nice to have: In error messages, it's nice to specify what they should do to fix it. Just adding "Please choose another time or move {name of event}" would be a nice touch.

Reply:
good idea, added to nice to haves